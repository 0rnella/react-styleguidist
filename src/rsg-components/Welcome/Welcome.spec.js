import React from 'react';
import { WelcomeRenderer } from './WelcomeRenderer';

it('renderer should welcome screen', () => {
	const actual = shallow(
		<WelcomeRenderer classes={{}} />
	);

	expect(actual).toMatchSnapshot();
});
