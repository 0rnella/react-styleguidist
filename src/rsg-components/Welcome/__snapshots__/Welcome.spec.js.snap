// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renderer should welcome screen 1`] = `
<div>
  <_class
    text="
  # Welcome to React Styleguidist!
  
  ## Point Styleguidist to your React components
  
  We couldnâ€™t find any components defined in your style guide config or using the default pattern \`src/components/**/*.js\`.
  
  Create **styleguide.config.js** file in your project root directory like this:
  
  module.exports = {
    components: 'src/components/**/*.js'
  };
  
  Read more in the [locating components guide](https://github.com/styleguidist/react-styleguidist/blob/master/docs/Components.md).
  				"
  />
</div>
`;
